### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by yamakataoka**
just fix typos

thank you.

### CODE REVIEW

File: doc/terms_x-road_docs.md [hunk 1]
```diff
@@ -20,23 +20,23 @@ Document ID: TA-TERMS
 - [2 Participants of X-Road](#2-participants-of-x-road)
 - [3 Trust services](#3-trust-services)
 - [4 Roles of X-Road member](#4-roles-of-x-road-member)
-  * [4.1 In terms of dataservice](#41-in-terms-of-dataservice)
-  * [4.2 In terms of management of security server](#42-in-terms-of-management-of-security-server)
+* [4.1 In terms of dataservice](#41-in-terms-of-dataservice)
```
Comment (petkivim) [inline doc/terms_x-road_docs.md]: Subsection-headings must be indented. Please restore the original indentation. This comment is applicable to subsection headings 4.x, 6.x and 7.x.

File: doc/terms_x-road_docs.md [hunk 2]
```diff
@@ -181,8 +181,8 @@ This document is licensed under the Creative Commons Attribution-ShareAlike 3.0
 - **Environmental monitoring** – is the monitoring of the X-Road environment: details of the security servers such as operating system, memory, disk space, CPU load, running processes and installed packages, etc.
 
 - **Operational monitoring** – is the monitoring of operational statistics such as which services have been called, how many times, what is the average response time, etc.
-    + **Operational monitoring data** – contains operational data (such as which services have been called, how many times, what was the size of the response, etc.) of the X-Road security server(s).
-    + **Operational monitoring daemon** – collects and shares operational monitoring data of the X-Road security server(s), calculates and shares health data of the X-Road security server(s) that is based on collected operational monitoring data.
++ **Operational monitoring data** – contains operational data (such as which services have been called, how many times, what was the size of the response, etc.) of the X-Road security server(s).
```
Comment (petkivim) [inline doc/terms_x-road_docs.md]: This section must be indented. Please restore the original indentation.

File: doc/terms_x-road_docs.md [hunk 3]
```diff
@@ -181,8 +181,8 @@ This document is licensed under the Creative Commons Attribution-ShareAlike 3.0
...
++ **Operational monitoring daemon** – collects and shares operational monitoring data of the X-Road security server(s), calculates and shares health data of the X-Road security server(s) that is based on collected operational monitoring data.
```
Comment (petkivim) [inline doc/terms_x-road_docs.md]: This section must be indented. Please restore the original indentation.

**Comment by yamakataoka** [comment]
Thank you for your review
I'm so sorry, that's my bad.

My fixing: capitalization/ punctuation mark/ certificate

**Comment by petkivim** [comment]
@yamakataoka Thank you for the contribution!

**Pull request merged**